go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/moby/buildkit
 
-go 1.21
+go 1.21.0
```

Comment:
yikes: https://github.com/moby/buildkit/actions/runs/9779428021/job/26998669203?pr=5118#step:5:539

we need to set `go 1.21.0` in our `go.mod`: https://github.com/gofrs/flock/blob/5169644b9198f307ed198a9360aab196cbd07409/go.mod#L3 :expressionless: